File: median-array.py
Array1 = list(range(1, 2))
Array2 = list(range(2, 5))


def median_array(arr1, arr2):
    combined_array = arr1 + arr2
    sortedarray = sorted(combined_array)

    mid_value = len(sortedarray) // 2
    if len(sortedarray) % 2 == 0:
        median = (sortedarray[mid_value - 1] + sortedarray[mid_value]) / 2
        print("Median of the array is:", median)
    else:
        print("Median of the array is:", sortedarray[mid_value])


median_array(Array1, Array2)

File: median-array.js
Array1 = [1, 2, 3, 4];
Arary2 = [5, 6, 7, 8];

function findMedianSortedArrays(nums1, nums2) {
  let combined_array = nums1.concat(nums2);
  sorted_Array = combined_array.sort((a, b) => a - b);
  let length_array = sorted_Array.length;
  mid_value = length_array / 2;
  if (length_array % 2 === 0) {
    console.log(
      `${(sorted_Array[mid_value - 1] + sorted_Array[mid_value]) / 2}`
    );
  } else {
    console.log(`${sorted_Array[Math.floor(mid_value)]}`);
  }
}
findMedianSortedArrays(Array1, Arary2);
